src/plugins/intel_gpu/src/graph/batch_to_space.cpp

```diff
@@ -66,6 +68,94 @@ layout batch_to_space_inst::calc_output_layout(batch_to_space_node const& node,
     return layout{output_type, input_format, desc->out_size};
 }
 
+static std::vector<int32_t> tensorToVec(const tensor& t, const format f) {
+    std::vector<int32_t> vec(cldnn::format::dimension(f));
+    for (size_t i = 0; i < vec.size(); ++i) {
+        vec[i] = t.sizes()[i];
+    }
+    std::reverse(vec.begin() + 2, vec.end());
+    return vec;
+}
+
+template<typename ShapeType>
+std::vector<layout> batch_to_space_inst::calc_output_layouts(batch_to_space_node const& /*node*/, const kernel_impl_params& impl_param) {
+    auto desc = impl_param.typed_desc<batch_to_space>();
+    auto input0_layout = impl_param.get_input_layout(0);
+    auto input0_shape = input0_layout.get<ShapeType>();
+    auto input0_size = input0_layout.get_partial_shape().size();
+    auto input0_format = input0_layout.format;
+
+    auto& constant_mem = impl_param.memory_deps;
+    auto block_data = desc->block_shape;
+    auto begin_data = desc->crops_begin;
+    auto end_data = desc->crops_end;
+
+    if (desc->shape_constant == 0 && (!constant_mem.count(1) || !constant_mem.count(2) || !constant_mem.count(3))) {
+        auto out_shape = ov::PartialShape::dynamic(input0_layout.get_partial_shape().size());
+        return { layout{out_shape, input0_layout.data_type, format::get_default_format(out_shape.rank().get_length())} };
+    }
```

Comment:
We should not change the format at calc_output_layout, basically format should be selected by get_preferred_format. 
Please just use input0 format.

Reply:
done